### PR TITLE
[Fused] Unsqueeze op

### PR DESCRIPTION
**Pull request by postrational**
Adding fused op checklist:
- [x] Update https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/op/fused_op_tbl.hpp file.
- [x] Update serializer: https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/serializer.cpp
- [x] Add UT here: https://github.com/NervanaSystems/ngraph/blob/master/test/backend_fused_op.in.cpp
- [x] Add simple UT here: https://github.com/NervanaSystems/ngraph/blob/master/test/type_prop.cpp#L13364
- [x] Add ops to aggregate nGraph header: https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/ngraph.hpp#L98
- [x] Update https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/runtime/intelgpu/intelgpu_backend.cpp
- [x] If UTs fail, update https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/runtime/gpu/unit_test.manifest
- [x] If UTs fail, update https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/runtime/intelgpu/unit_test.manifest

### CODE REVIEW

File: src/ngraph/frontend/onnx_import/op/unsqueeze.cpp [hunk 1]
```diff
@@ -35,31 +28,11 @@ namespace ngraph
...
-                            << "provided 'axes' attribute is not valid.";
-
-                        data_shape.insert(std::next(std::begin(data_shape), axis), 1);
-                    }
-
-                    return {std::make_shared<ngraph::op::Reshape>(data, input_order, data_shape)};
+                    auto data = node.get_ng_inputs().at(0);
+                    auto axes = node.get_attribute_value<std::vector<std::int64_t>>("axes", {});
+                    auto axes_node = std::make_shared<ngraph::op::Constant>(
+                        element::u64, Shape{axes.size()}, axes);
```
Comment (tsocha) [inline src/ngraph/frontend/onnx_import/op/unsqueeze.cpp]: Is it intended to use `unsigned int` as element type for `axes_node` while `axes` is `signed int`?

File: src/ngraph/op/fused/unsqueeze.cpp [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+NodeVector op::Unsqueeze::decompose_op() const
+{
+    auto data = get_argument(0);
+    auto axes_node = get_argument(1);
+
+    // Currently only support Constant node for axes.
+    NODE_VALIDATION_CHECK(this,
+                          axes_node->description() == "Constant",
```
Comment (tsocha) [inline src/ngraph/op/fused/unsqueeze.cpp]: Please use `is_constant` method from `Node` class.

File: src/ngraph/op/fused/unsqueeze.cpp [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    // Get value of axes from Constant
+    auto axes_constant = dynamic_pointer_cast<op::Constant>(axes_node);
+    auto axes = axes_constant->get_vector<size_t>();
+
+    auto data_shape = data->get_shape();
+
+    NODE_VALIDATION_CHECK(this, !axes.empty(), "'axes' input is mandatory.");
+    NODE_VALIDATION_CHECK(this,
+                          axes.size() ==
+                              std::set<std::int64_t>(std::begin(axes), std::end(axes)).size(),
```
Comment (arogowie-intel) [inline src/ngraph/op/fused/unsqueeze.cpp]: Please remove `std::` prefixes.

File: src/ngraph/serializer.cpp [hunk 4]
```diff
@@ -78,6 +78,7 @@
 #include "ngraph/op/fused/prelu.hpp"
 #include "ngraph/op/fused/scale_shift.hpp"
 #include "ngraph/op/fused/space_to_depth.hpp"
+#include "ngraph/op/fused/unsqueeze.hpp"
```
Comment (dmyershov) [inline src/ngraph/serializer.cpp]: Please add `case OP_TYPEID::Unsqueeze:` line between 2040 - 2081 lines and between 2161 - 2171 lines.

File: src/ngraph/op/fused/unsqueeze.cpp [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+NodeVector op::Unsqueeze::decompose_op() const
+{
+    auto data = get_argument(0);
+    auto axes_node = get_argument(1);
+
+    // Currently only support Constant node for axes.
+    NODE_VALIDATION_CHECK(this,
+                          axes_node->is_constant(),
+                          "doesn't support 'axes' input of other type than a Constant.");
```
Comment (aprocter) [inline src/ngraph/op/fused/unsqueeze.cpp]: This makes me a bit nervous, but we will soon have a better story here so I think it's okay to impose this restriction for now.

**Pull request merged**